src/nbl/builtin/utils.cmake

```diff
@@ -8,14 +12,20 @@ macro(LIST_BUILTIN_RESOURCE _BUNDLE_NAME_ _LBR_PATH_)
 	set(_LBR_${_BUNDLE_NAME_}_ ${_LBR_${_BUNDLE_NAME_}_} PARENT_SCOPE) # override
 endmacro()
 
-function(ADD_CUSTOM_BUILTIN_RESOURCES _TARGET_NAME_ _BUNDLE_NAME_ _BUNDLE_SEARCH_DIRECTORY_ _NAMESPACE_ _OUTPUT_DIRECTORY_)
+# Creates a library with builtin resources for given bundle
+# _TARGET_NAME_ is name of a target library that will be created
+# _BUNDLE_NAME_ is a bundle name builtin resources are associated with
+# _BUNDLE_SEARCH_DIRECTORY_ is an absolute search directory path for builtin resorces for given bundle
+# _NAMESPACE_ is a C++ namespace builtin resources will be wrapped into
+# _OUTPUT_INCLUDE_SEARCH_DIRECTORY_ is an absolute path to output directory for builtin resources header files which will be a search directory for generated headers outputed to ${_OUTPUT_HEADER_DIRECTORY_}/${_PATH_PREFIX_} where path prefix is the namespace turned into a path
```

Comment:
should be called `_OUTPUT_HEADER_DIRECTORY_`

Reply:
`_OUTPUT_HEADER_DIRECTORY_` is different then `_OUTPUT_INCLUDE_SEARCH_DIRECTORY_`